### PR TITLE
FEATURE: [Okex] Add order and trade methods for okex

### PR DESCRIPTION
**Pull request by MengShue**
1. Add QueryTrades() , QueryClosedOrders(), QueryOrderTrades() 
2. little refactor okex

### CODE REVIEW

**Comment by ycdesu** [comment]
This is a huge pull request, and we change  about 1,500 lines of code in this pull request. It's difficult to review so many changes within one single pull request. Could you split this pull request into smaller ones?

File: pkg/exchange/okex/okexapi/trade.go [hunk 1]
```diff
@@ -430,12 +427,13 @@ func (r *GetPendingOrderRequest) Parameters() map[string]interface{} {
 
 func (r *GetPendingOrderRequest) Do(ctx context.Context) ([]OrderDetails, error) {
 	payload := r.Parameters()
-	req, err := r.client.newAuthenticatedRequest("GET", "/api/v5/trade/orders-pending", nil, payload)
+	fmt.Println(payload)
```
[the remark this answers is not in the document]
Comment (MengShue) [reply pkg/exchange/okex/okexapi/trade.go]: Yes, Sorry,  Updated

File: pkg/exchange/okex/okexapi/get_order_history_request.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package okexapi
+
+import (
+	"time"
+
+	"github.com/c9s/requestgen"
+)
+
+//go:generate -command GetRequest requestgen -method GET -responseType .APIResponse -responseDataField Data
+//go:generate -command PostRequest requestgen -method POST -responseType .APIResponse -responseDataField Data
```
[the remark this answers is not in the document]
Comment (MengShue) [reply pkg/exchange/okex/okexapi/get_order_history_request.go]: Actually I just copied from other files, and after I tested, there are no need to add these 2 lines, I remove it (and update file name to fit requestgen)

**Comment by MengShue** [comment]
> This is a huge pull request, and we change about 1,500 lines of code in this pull request. It's difficult to review so many changes within one single pull request. Could you split this pull request into smaller ones?

I wish I could, but I don't have a good commit to split this PR into small ones, ( https://stackoverflow.com/questions/30768148/git-split-pull-request-into-smaller-prs-based-upon-the-new-directories-in-the ) it really means that I have to write it again, or Do you have some git tricks to do "split PR"
I will keep in mind that I should have good break point to add commits and don't delivery such huge PR.

**Pull request closed without merging**